examples/tot_crosswords/prompts/crosswords.py

```diff
@@ -0,0 +1,326 @@
+# 5 shot

```

Comment:
What's the difference from previous json file?

Reply:
we used this one for prompts. I removed the json file